### ISSUE TITLE
Learn cloud-native Java development with Jakarta EE & MicroProfile in Japanese

### ISSUE THREAD

**Issue by yeekangc** [OpenLiberty/blogs#2351]
Consider a blog post that talks about how we can use the Open Liberty guides to learn about cloud-native Java development with Jakarta EE and MicroProfile in Japanese.  @una-tapa @takakiyo have an approach that works rather nicely and easily.  (Thank you!)

Use this to work through handling non-English blog posts on opneliberty.io too.

Cc @kinueng @lauracowen @GraceJansen @dmuelle

**Comment by yeekangc** [comment]
@una-tapa, feel free to take ownership of this issue.  =)  Thanks.

**Comment by una-tapa** [comment]
Hello,reviewers (@yeekangc@kinueng @lauracowen @GraceJansen @dmuelle) 

I will appreciate your review for the following draft.  The actual article will be in Japanese. It is a translated version to get your review on the contents. Suggestions and corrections will be really appreciated.
 
https://github.com/OpenLiberty/blogs/blob/japanese-blog-test/posts/2022-09-01-openliberty-guides-in-japaneseE.adoc

Takakiyo-san @takakiyo

Please review the Japanese version when you get a chance. I agree that the Chrome UI looks a bit different between Japanese version and the U.S version. I will appreciate some help with the screenshot. 

https://github.com/OpenLiberty/blogs/blob/japanese-blog-test/posts/2022-09-01-openliberty-guides-in-japanese.adoc

**Comment by kinueng** [comment]
Hi @una-tapa,
Very exciting to have our first multi-language blog post!

This week we merged our initial code to support non English content to openliberty.io. I will run with this new Japanese `adoc` file and use it with our new multi-language code.

I see the blog post gives some instructions on how to use browser machine translation technology https://openliberty.io/guides/. This may change as we think through how to display posts in different languages.

(fyi @Sreejith-Websphere)

**Comment by lauracowen** [comment]
Hiya @una-tapa 

Looks good, nice and simple and clear. The one comment I had was where it says "The contents of the Guides have tags around code and commands. This helps the translation to be only applied to the explanations leaving the code and commands unchanged. "

* The first sentence seemed like something a user of the guides shouldn't care about (how we construct/build/translated the guides). 
* The second sentence also seemed kind of obvious to tech people used to using translated products (that the commands and code aren't translated), but maybe commands do sometimes get translated and I just didn't know that? In which case, this probably only needs to be mentioned once, rather than twice, in the blog post?

**Comment by una-tapa** [comment]
@kinueng  That's exciting to hear that multi-language support is getting in. Please keep me posted on the result after running it with Japanese adoc. If it breaks, I will adjust adoc.  I will try the instruction again (and ask Japan team to try also) after the support went in. 

@lauracowen  Wow, great suggestion.  I will remove these lines. . 

Just a background, We found Liberty Guides get translated by the browser tool exceptionally well compared with other similar articles. We happened to find out the tags in the Guides helping a lot. I was excited to share... among us.... I totally agree with you the readers do not care at all. The  two sentences are cluttering the blog. I will remove it.  Thanks.

**Comment by lauracowen** [comment]
@una-tapa Oh wow, that's really good. If you have any other tips, the OL docs team might be interested (if not already involved). cc/ @dmuelle 

Thanks for all your help with this!

**Comment by kinueng** [comment]
@una-tapa I misunderstood the intent of the section showcasing the use of the browser to machine translate the entire catalog of guides. I had thought the section was suggested as a way to switch between blogs that are written in English and Japanese. 

Rather, this whole blog post is to help Japanese readers to consume all our entire catalog if the machine translation from Google is good. Please disregard my suggestion to remove anything. I should of read the whole blog post before commenting 😬.

My mind is deep in thinking how we can show this blog post that is written in Japanese (and not machine translated) on the website.

**Comment by una-tapa** [comment]
Fixed by #2534 

@kinueng 
I understand that your mind are deep in thinking to see this fits into the current blog structure.  I guess this blog can be used to explore... and if you decide it is not a good time to publish non-English blog, that's fine with me too. 
 
Today, Takakiyo-san found the time to review the contents and gave it OK. So I decided to submit the following PRs to finish my part. 

- The Japanese version #2534 (Only Japanese audience in mind) 
- The English version #2535 (having non-English readers in mind) 

Thanks!
